src/test/resources/network/brightspots/rcv/test_data/hart_cedar_park_school_board/hart_cedar_park_school_board_config.json

```diff
@@ -0,0 +1,80 @@
+{
+  "tabulatorVersion" : "1.1.0",
+  "outputSettings" : {
+    "contestName" : "CEDAR PARK SCHOOL BOARD",
+    "contestId" : "b651b997-417a-46d9-a676-a43d4df94ddc",
+    "outputDirectory" : "output",
+    "contestDate" : "",
+    "contestJurisdiction" : "",
+    "contestOffice" : "",
+    "tabulateByPrecinct" : false,
+    "generateCdfJson" : false
+  },
+  "cvrFileSources" : [ {
+    "filePath" : "CVRArchive_18_9_0_6298/",
```

Comment:
This test fails for me with:

> 2020-07-08 12:20:54 PDT INFO: Reading Hart cast vote record file: 04917dca-3f9e-440e-a41a-98dc5e57376e.xml...
> 2020-07-08 12:20:54 PDT SEVERE: Error parsing cast vote record:
com.fasterxml.jackson.core.JsonParseException: Unexpected first character (char code 0xEF), not valid in xml document: could be mangled UTF-8 BOM marker. Make sure that the Reader uses correct encoding or pass an InputStream instead
> 2020-07-08 12:20:54 PDT SEVERE: Error opening cast vote record file: C:\Users\UserName\IdeaProjects\rcv\src\test\resources\network\brightspots\rcv\test_data\hart_cedar_park_school_board\CVRArchive_18_9_0_6298
> 2020-07-08 12:20:54 PDT INFO: Check file path and permissions and make sure they are correct!